src/components/common/overflow-menu/index.tsx

```diff
@@ -0,0 +1,113 @@
+import Text from '../text';
+import Link from 'next/link';
+import { MdManageAccounts, MdLogout, MdAccountCircle } from 'react-icons/md';
+import { VscGithubAlt } from 'react-icons/vsc';
+import { BiWalletAlt } from 'react-icons/bi';
+import { useState, useRef } from 'react';
+import { cn } from 'utils';
+import Card from '../card';
+const OverflowMenu = props => {
```

Comment:
Please add a type definition here and adjust props accordingly.

Reply:
Ok I will do that